community/index.html

```diff
@@ -159,6 +159,8 @@ <h3 id="older-archives">Older Archives</h3>
 </ul>
 <h2 id="ozone-slack">Ozone Slack</h2>
 <p>If you would like to join the Ozone Slack Channel, please use this link to get self-invited  <a href="https://s.apache.org/slack-invite">https://s.apache.org/slack-invite</a> and join the <strong>#ozone</strong> channel.</p>
+<h2 id=github-discussions">GitHub Discussions</h2>
+<p><a href="https://github.com/apache/ozone/discussions">GitHub Discussions</a> is an alternative to Slack and only needs a GitHub account to interact with the community</p>
 <h2 id="ozone-community-calls">Ozone Community Calls</h2>
 <p>Please refer to <a href="https://cwiki.apache.org/confluence/display/OZONE/Ozone+Community+Calls">this page</a> to get more information.</p>
```

Comment:
@kerneltime @smengcl Ozone website source is in the `master` branch of this repo.  Contents in `asf-site` branch are generated.  Your changes made here will be overwritten next time someone modifies the source.  Please add the new section in:

https://github.com/apache/ozone-site/blob/3605743c470579964538b58732145c61a9f41656/content/community.md?plain=1#L45-L51